cmd/src/config.go

```diff
@@ -0,0 +1,186 @@
+	edit      updates configuration settings
+	list      lists the partial settings (that, when merged, yield the effective configuration)
+
+Use "src config [command] -h" for more information about a command.
+`
+
+	flagSet := flag.NewFlagSet("config", flag.ExitOnError)
+	handler := func(args []string) error {
+		configCommands.run(flagSet, "src config", usage, args)
+		return nil
+	}
+
+	// Register the command.
+	commands = append(commands, &command{
+		flagSet: flagSet,
+		handler: handler,
+		usageFunc: func() {
+			fmt.Println(usage)
+		},
+	})
+}
+
+const configurationSubjectFragment = `
+fragment ConfigurationSubjectFields on ConfigurationSubject {
+    id
+    latestSettings {
+        id
+        configuration {
+            ...ConfigurationFields
+        }
+        author {
+            ...UserFields
+        }
+        createdAt
+    }
+    settingsURL
+    viewerCanAdminister
+}
+`
+
+type ConfigurationSubject struct {
+	ID                   string
+	LatestSettings       *Settings
+	SettingsURL          string
+	ViewerCanAdminister  bool
+	ConfigurationCascade ConfigurationCascade
+}
+
+type Settings struct {
+	ID            int32
+	Configuration Configuration
+	Author        *User
+	CreatedAt     string
+}
+
+const configurationCascadeFragment = `
+fragment ConfigurationCascadeFields on ConfigurationCascade {
+    subjects {
+        ...ConfigurationSubjectFields
+    }
+    merged {
+        ...ConfigurationFields
+    }
+}
+`
+
+type ConfigurationCascade struct {
+	Subjects []ConfigurationSubject
+	Merged   Configuration
+}
+
+const configurationFragment = `
+fragment ConfigurationFields on Configuration {
+    contents
+    messages
+}
+`
+
+type Configuration struct {
+	Contents string
+	Messages []string
+}
+
+const viewerConfigurationQuery = `query ViewerConfiguration {
+  viewerConfiguration {
+    ...ConfigurationCascadeFields
+  }
+}` + configurationCascadeFragment + configurationSubjectFragment + configurationFragment + userFragment
+
+const configurationSubjectCascadeQuery = `query ConfigurationSubjectCascade($subject: ID!) {
+  configurationSubject(id: $subject) {
+    configurationCascade {
+      ...ConfigurationCascadeFields
+    }
+  }
+}` + configurationCascadeFragment + configurationSubjectFragment + configurationFragment + userFragment
+
+type KeyPath struct {
+	Property string `json:"property,omitempty"`
+	Index    int    `json:"index,omitempty"`
+}
+
+func getViewerUserID() (string, error) {
+	var result struct {
+		CurrentUser *struct{ ID string }
+	}
+	req := &apiRequest{
+		query: `
+query ViewerUserID {
+  currentUser {
+    id
+  }
+}
+`,
+		result: &result,
+	}
+	if err := req.do(); err != nil {
+		return "", err
+	}
+	if result.CurrentUser == nil || result.CurrentUser.ID == "" {
+		return "", errors.New("unable to determine current user ID (ensure the access token you're using to authenticate the CLI is correct, or provide one if none exists; see 'src -h' for more information)")
```

Comment:
In other locations we link to the README for how to authenticate actually, which helps a bit with shortening the wording a bit. See https://gotools.org/github.com/sourcegraph/src-cli/cmd/src#api.go-L220

Reply:
Thanks, pushed a new commit that links to the README